code/datums/ai/hunting_behavior/hunting_behaviors.dm

```diff
@@ -85,7 +79,10 @@
 /datum/ai_behavior/hunt_target/setup(datum/ai_controller/controller, hunting_target_key, hunting_cooldown_key)
 	. = ..()
 	var/datum/weakref/hunting_weakref = controller.blackboard[hunting_target_key]
-	controller.set_movement_target(hunting_weakref?.resolve())
+	var/atom/hunt_target = hunting_weakref?.resolve()
+	if (QDELETED(hunt_target))
```

Comment:
```suggestion
	if (isnull(hunt_target))
```

Reply:
This pattern seems to be all over ai behaviours unfortunately 🙃